example/docker-compose.extra.yml

```diff
@@ -29,7 +29,7 @@ services:
          - elasticsearch.svc
     environment:
       discovery.type: single-node
-      ES_JAVA_OPTS: "-Xms512m -Xmx512m -Dlog4j2.formatMsgNoLookups=true"
+      ES_JAVA_OPTS: "-Xms512m -Xmx512m"
```

Comment:
After sleeping on it, lets keep this in because some people might still be running versions where this isn't in the container.

Reply:
I agree. Out of abundance of caution, we shall keep the setting in, even if it is redundant for the "usual" use case (i.e. running the container version which already has the setting on,